lib/brainstem/presenter_collection.rb

```diff
@@ -183,5 +191,17 @@ def check_for_old_options!(options)
         raise "PresenterCollection#presenting no longer accepts the :as option.  Use the brainstem_key annotation in your presenters instead."
       end
     end
+
+    def page_number(count, params)
+      count > 0 ? params.fetch(:page, 1).to_i : 0
+    end
+
+    def page_count(count, per_page)
+      count > 0 ? (count.to_f / per_page).ceil : 0
+    end
+
+    def per_page(params)
+      [1, params.fetch(:per_page, default_per_page).to_i].max
```

Comment:
Hmm... wonder if we need to be considering the max per page here, as well

Reply:
That's a good call. I would think so. The should be a getter for that. We would need to reference it above as well.

Reply:
Nice catch. If we haven't already, we certainly want to guard against this.